platform/server/src/mongo/createMongoServices.ts

```diff
@@ -2,7 +2,7 @@ import { GraphQLObjectType, GraphQLSchema } from "graphql"
 import { PubSubEngine } from 'graphql-subscriptions';
 import { CRUDService, GraphbackPubSubModel } from "@graphback/runtime"
 import { Db } from "mongodb"
-import { MongoDBDataProvider } from "./MongoDBDataProvider";
+import { OffixMongoDBDataProvider } from '@graphback/runtime-mongo';
```

Comment:
FYI - we kept our own data provider to drive some requirements. 
We might need to change some things to OffixMongoDBProvider later and use id directly here but for now is fine :D

Reply:
I was curious why the `version` field was not set when I created a recipient, product etc. Since we have offline capabilities in client app, I thought we must bring this.